src/schema.ts

```diff
@@ -7,15 +8,40 @@ const urlStart = 'https://';
 const urlEnd = '.okta.com';
 export const urlSchema = z
   .string()
-  .url()
-  .startsWith(urlStart, `URL must start with [${urlStart}].`)
-  //.co will cause fetch to time out
-  .endsWith(urlEnd, `URL must end with [${urlEnd}].`)
-  .min(
-    urlStart.length + urlEnd.length + 1,
-    'Domain name must be at least 1 character long.'
-  )
+  // eslint-disable-next-line @typescript-eslint/prefer-readonly-parameter-types
+  .transform((urlString, ctx) => {
+    // eslint-disable-next-line functional/no-conditional-statement
+    if (!urlString.startsWith(urlStart)) {
```

Comment:
**Nit:** this could be replaced with a post-transform refinement that checks that `url.protocol = 'https'` which is a cleaner way of expressing this requirement.

Reply:
This won't check for the double / requirement.

Reply:
If `URL` (and consequentially `ReadonlyURL`) parses `https:/blahblah` into:

```
{
   host: blahblah,
   protocol: https
}
```
And normalises the output as `https://blahblah`, do we care whether the user provided one or two slashes?